src/actions.rs

```diff
@@ -93,7 +119,16 @@ impl<'a> Action for Step<'a> {
                                     })
                                     .collect();
 
-                                context.insert(*name, &issues_decorator);
+                                // group query results for multiline FCP and add them later
+                                if PENDINGFCP.contains(name) {
+                                    all_pending_fcp.push(("in_pre_fcp", issues_decorator));
+                                } else if THINGSINFCP.contains(name) {
+                                    all_things_in_fcp.push(("in_fcp", issues_decorator));
+                                } else if FINALIZEDFCP.contains(name) {
+                                    all_finalized_fcp.push(("fcp_finished", issues_decorator));
```

Comment:
Wouldn't we also want constants for these ones?, "in_pre_fcp", "in_fcp" and "fcp_finished"

Reply:
why not :+1: although they're only used in the template